app/controllers/nonsipp/membersurrenderedbenefits/SurrenderedBenefitsMemberListController.scala

```diff
@@ -130,31 +128,30 @@ class SurrenderedBenefitsMemberListController @Inject()(
     val userAnswersWithSurrenderedBenefitsMemberList =
       request.userAnswers.set(SurrenderedBenefitsMemberListPage(srn), selection)
 
-    //TODO: complete once Surrendered Benefits journey is complete/as part of transformation work
-    Future.fromTry(userAnswersWithSurrenderedBenefitsMemberList)
-//    if (selection) {
-//      val indexes = (1 to memberListSize)
-//        .map(i => refineV[OneTo300](i).leftMap(new Exception(_)).toTry)
-//        .toList
-//        .sequence
-//
-//      Future.fromTry(
-//        indexes.fold(
-//          _ => userAnswersWithSurrenderedBenefitsMemberList,
-//          index =>
-//            index.foldLeft(userAnswersWithSurrenderedBenefitsMemberList) {
-//              case (userAnswersTry, index) =>
-//                val optSurrenderedBenefitsAmount = request.userAnswers.get(SurrenderedBenefitsAmountPage(srn, index))
-//                for {
-//                  ua <- userAnswersTry
-//                  ua1 <- ua.set(SurrenderedBenefitsAmountPage(srn, index), optSurrenderedBenefitsAmount.getOrElse(Money(0)))
-//                } yield ua1
-//            }
-//        )
-//      )
-//    } else {
-//      Future.fromTry(userAnswersWithSurrenderedBenefitsMemberList)
-//    }
+    if (selection) {
```

Comment:
Same here, @narendrapaduchuri and @ccatosdevelopment please cross check this is still right